airflow-ctl/pyproject.toml

```diff
@@ -26,7 +26,8 @@ license-files = ["LICENSE", "NOTICE"]
 # out-of-the box. Airflow-ctl is a small tool that does not have many dependencies and does not use
 # sophisticated features of Python, so it should work with Python 3.14+ once all it's dependencies are
 # updated to support it.
-requires-python = ">=3.10"
+
+requires-python = ">=3.10,<3.14"
```

Comment:
See https://github.com/apache/airflow/pull/58657 -> as surprising as it is < on python_requires is not really something recommended. It's recommended to explicitly exclude the versions we know are not supported. We had a long discussion about it in https://github.com/astral-sh/uv/issues/14711